common/annotations/ha.xml

```diff
@@ -2218,11 +2218,11 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		<annotation cp="🪢">igiya | karkata | ƙulla | ƙulli | ruɗani</annotation>
 		<annotation cp="🪢" type="tts">ƙulli</annotation>
 		<annotation cp="👓">ido | kayan ido | sutura | tabarau</annotation>
-		<annotation cp="👓" type="tts">tabarau</annotation>
+		<annotation cp="👓" type="tts">gilashin ido/annotation>
```

Comment:
This needs a </ instead of a <

This causes an error in reading ha.xml

Reply:
```suggestion
		<annotation cp="👓" type="tts">gilashin ido</annotation>
```